M2/Macaulay2/m2/basictests/regex.m2

```diff
@@ -1,19 +0,0 @@
-assert = x -> if not x then error "assertion failed "
-match = X -> null =!= regex X
-assert not match(".a", "  \na  ")
-assert     match("^a", "  \na  ")
-assert     match(".a", "  a  ")
-assert not match("^a", "  a  ")
-assert match("a|b","a")
-assert match("a+","a")
-assert match("(a)","a")
-assert match("a{2}","aa")
-assert match("a[2]","a2")
-assert match("a[^a]","a2")
-assert match("a\\b","a b")
-assert not match("a\\b","ab")
-assert match("a\\>","a b")
-assert match("a\\>","a")
-assert not match("a\\>"," ab")
-
-assert( replaceStrings("^a","x","a \na \naaa a") === "x \nx \nxaa a" )
```

Comment:
Why was this file deleted?  Or are the tests now somewhere else?

Reply:
They're not deleted, simply moved to `tests/normal/regex.m2`.